src/server/system_services/stats_aggregator.js

```diff
@@ -972,6 +972,9 @@ async function object_usage_scrubber(req) {
 //_.noop(send_stats_payload); // lint unused bypass
 
 function send_stats_payload(payload) {
```

Comment:
Notice that this function was an async function (using Promise) since we did not have any returns and it started from:
```return P.fromCallback(callback => request(options, callback)```
The addition of return doesn't change or create any bugs in our current usage because we wrap the call in a Promise like this:
```
return P.resolve()
        .then(() => {
            if (DEV_MODE) {
                dbg.log('Central Statistics payload send is disabled in DEV_MODE');
                return P.resolve();
            }
            return send_stats_payload(payload);
        })
```
But if a person would want to use this function without wrapping it won't return a Promise and will give an exception.
I would advise to re-write the function using Async/Await or return a Promise instead of undefined.